controllers/networkchaos/partition/types.go

```diff
@@ -253,32 +253,30 @@ func (e *endpoint) SetChains(ctx context.Context, pods []v1.Pod, chains []v1alph
 	return nil
 }
 
-// Recover recovers the chaos
+// Recover means the reconciler recovers the chaos action
 func (e *endpoint) Recover(ctx context.Context, req ctrl.Request, chaos v1alpha1.InnerObject) error {
 	networkchaos, ok := chaos.(*v1alpha1.NetworkChaos)
 	if !ok {
 		err := errors.New("chaos is not NetworkChaos")
 		e.Log.Error(err, "chaos is not NetworkChaos", "chaos", chaos)
-
 		return err
 	}
 
 	if err := e.cleanFinalizersAndRecover(ctx, networkchaos); err != nil {
-		e.Log.Error(err, "cleanFinalizersAndRecover failed")
 		return err
 	}
 	e.Event(networkchaos, v1.EventTypeNormal, events.ChaosRecovered, "")
 
 	return nil
 }
 
-func (e *endpoint) cleanFinalizersAndRecover(ctx context.Context, networkchaos *v1alpha1.NetworkChaos) error {
+func (e *endpoint) cleanFinalizersAndRecover(ctx context.Context, chaos *v1alpha1.NetworkChaos) error {
```

Comment:
Here is also the repeated logic. Can we improve this logic?

Reply:
The current implementation makes use of the same variables in 'recover' of different chaos. But for `networkchaos` and `iochaos` which use its own `chaosmanager`, things could not be solved in this way. @YangKeao [suggested](https://github.com/chaos-mesh/chaos-mesh/pull/1014#issuecomment-747188298) to leave this part away for now, until a common [`subresourcemanager`](https://github.com/chaos-mesh/chaos-mesh/pull/1014#issuecomment-706519681) is implemented